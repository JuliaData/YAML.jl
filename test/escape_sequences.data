escape sequences:
- '\0': "\0"
- '\a': "\a"
- '\b': "\b"
- '\t': "\t"
- '\n': "\n"
- '\v': "\v"
- '\f': "\f"
- '\r': "\r"
- '\e': "\e"
- '\ ': "\ "
- '\"': "\""
- '\\': "\\"
- '\N': "\N"
- '\_': "\_"
- '\L': "\L"
- '\P': "\P"
