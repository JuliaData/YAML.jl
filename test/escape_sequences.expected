Dict{Any,Any}(Pair{Any,Any}("escape sequences", Dict{Any,Any}[Dict{Any,Any}(Pair{Any,Any}("\\0", "\0")), Dict{Any,Any}(Pair{Any,Any}("\\a", "\a")), Dict{Any,Any}(Pair{Any,Any}("\\b", "\b")), Dict{Any,Any}(Pair{Any,Any}("\\t", "\t")), Dict{Any,Any}(Pair{Any,Any}("\\n", "\n")), Dict{Any,Any}(Pair{Any,Any}("\\v", "\v")), Dict{Any,Any}(Pair{Any,Any}("\\f", "\f")), Dict{Any,Any}(Pair{Any,Any}("\\r", "\r")), Dict{Any,Any}(Pair{Any,Any}("\\e", "\e")), Dict{Any,Any}(Pair{Any,Any}("\\ ", " ")), Dict{Any,Any}(Pair{Any,Any}("\\\"", "\"")), Dict{Any,Any}(Pair{Any,Any}("\\\\", "\\")), Dict{Any,Any}(Pair{Any,Any}("\\N", "\u85")), Dict{Any,Any}(Pair{Any,Any}("\\_", "Â ")), Dict{Any,Any}(Pair{Any,Any}("\\L", "\u2028")), Dict{Any,Any}(Pair{Any,Any}("\\P", "\u2029"))]))